support/core_test.go

```diff
@@ -0,0 +1,51 @@
+package support
+
+import (
+	"testing"
+
+	"github.com/onsi/gomega"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes/fake"
+)
+
+func TestGetPods(t *testing.T) {
+	// Create a fake Kubernetes client for testing
+	fakeClient := fake.NewSimpleClientset(&corev1.Pod{
```

Comment:
This can be replaced by `NewFakeKubeClientWithObjects`, right?

Reply:
Right Replaced